locales/bn/json.json

```diff
@@ -677,6 +677,7 @@
     "Qatar": "কাতার",
     "Quarter To Date": "তারিখ থেকে চতুর্থাংশ",
     "Quebec": "কুইবেক",
+    "Read": "পড়া",
```

Comment:
Don't add your keys to translation files as it's useless. The script checks if the keys match the set of Laravel packages and removes the extra ones.